### PR TITLE
refactor(utilities): EO-1152 Refactor rxLogout

### PR DESCRIPTION
**Pull request by manojdas2212**
JIRA:https://jira.rax.io/browse/EO-1152

### LGTMs
- [ ] Dev LGTM
- [ ] Dev+Vidyo LGTM
- [ ] Design LGTM
- ~~[ ] QE LGTM (no change to functional javascript)~~

Before
-----
screen shot 2016-09-16 at 5 24 17 pm

After
----
screen shot 2016-09-16 at 5 24 39 pm

API Docs
----
screen shot 2016-09-16 at 5 25 50 pm

All Modules
------
When filter by "logout"
----
screen shot 2016-09-16 at 5 25 21 pm

When filter by "rxlogout"
----
screen shot 2016-09-16 at 5 25 36 pm

### CODE REVIEW

File: src/utilities/rxLogout/docs/rxLogout.docs.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<p>
+  The rxLogout component provides logic to apply logout functionality to an element.
```
Comment (khaziwallis) [inline src/utilities/rxLogout/docs/rxLogout.docs.html]: @manojdas2212 it should be `The rxLogout utility...` i guess.

**Comment by Droogans** [comment]
Shouldn't rxLogout be deprecated by now? SAML doesn't understand logging out.

We've also dropped it from rxApp completely. It should be deleted, not moved.

File: src/utilities/rxLogout/docs/rxLogout.docs.html (hunk 1 above)
Comment (CITguy) [inline src/utilities/rxLogout/docs/rxLogout.docs.html]: agreed

**Comment by CITguy** [comment]
@Droogans: rxLogout cannot be removed in 2.x because it's a breaking change in the expected functionality.

File: src/utilities/rxLogout/rxLogout.meta.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "displayName": "rxLogout",
+    "stability": "stable",
```
Comment (CITguy) [inline src/utilities/rxLogout/rxLogout.meta.json]: Please update to `"deprecated"`. This utility is no longer useful and will be removed in version 3.0.0

File: src/utilities/rxLogout/scripts/rxLogout.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
-angular.module('encore.ui.rxLogout')
+angular.module('encore.ui.utilities')
 /**
  * @ngdoc directive
- * @name rxLogout.directive:rxLogout
+ * @name utilities.directive:rxLogout
  * @restrict A
```
Comment (CITguy) [inline src/utilities/rxLogout/scripts/rxLogout.js]: Please add `@deprecated` tag.

File: src/utilities/rxLogout/scripts/rxLogout.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
...
  * @scope
  * @description
```
Comment (CITguy) [inline src/utilities/rxLogout/scripts/rxLogout.js]: Please add the following to the top of the `@description` content:

```markdown
**DEPRECATED** This directive will be removed in version 3.0.0 on September 28, 2016.
```

**Comment by CITguy** [comment]
After thinking about it, this PR is moot. Since the documentation will be updated to 3.0.0 documentation on the next release, and because `rxLogout` will also be removed in 3.0.0, this will never be seen by consumers in the docs. I'm going to close this PR and its related JIRA story.

Thank you for your effort, but this is not needed.

**Pull request closed without merging**